iceoryx_binding_c/source/c_publisher.cpp

```diff
@@ -101,9 +101,9 @@ void iox_pub_deinit(iox_pub_t const self)
 
 iox_AllocationResult iox_pub_loan_chunk(iox_pub_t const self, void** const chunk, const uint32_t payloadSize)
 {
-    auto result = PublisherPortUser(self->m_portData).tryAllocateChunk(payloadSize).and_then([&](ChunkHeader* h) {
-        *chunk = h->payload();
-    });
+    auto result = PublisherPortUser(self->m_portData)
+                      .tryAllocateChunk(payloadSize, CHUNK_DEFAULT_PAYLOAD_ALIGNMENT)
+                      .and_then([&](ChunkHeader* h) { *chunk = h->payload(); });
```

Comment:
According to a very wise man (elBoberido the Great) you should only capture what you need.